app/controllers/application_controller.rb

```diff
@@ -107,7 +107,7 @@ def api_deprecation_error(exception = nil)
   def smart_proxy_exception(exception = nil)
     process_error(:redirect => :back, :error_msg => exception.message)
     logger.warn "ProxyAPI operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.warn exception.backtrace.join("\n")
```

Comment:
The more I think about it the more I think the silencer makes more sense in development, where usually the exceptions mean problems with your current changes (so the silencer makes it easier to find the issues quickly as you write the code), however in production, they are throwing away useful information. I'm ok with introducing a helper to log the exception: in that case I would revisit also the logging in orchestration, where we often throw away the backtrace https://github.com/theforeman/foreman/blob/6ace062d9dd9f0a529639d8123bf778072dce18e/app/models/concerns/orchestration/realm.rb#L17, so much for small PRs :)